chat-plugins/wifi.js

```diff
@@ -149,10 +149,10 @@ var LotteryGiveAway = (function () {
 		this.prize = options.prize;
 		this.maxwinners = options.maxwinners;
 		this.joined = {}; // userid: 0
-
+		var firstPart = this.prize[0];
 		this.reminder = '<center><div class = "broadcast-blue"><font size = 3><b>It\'s giveaway time!</b></font><br/>' +
 			'<font size = 1>Giveaway started by ' + Tools.escapeHTML(host.name) + '</font><br/><br/>' +
-			'<b>' + Tools.escapeHTML(giver.name) + '</b> will be giving away a <b>' + Tools.escapeHTML(this.prize) + '</b>!<br/>' +
+			'<b>' + Tools.escapeHTML(giver.name) + '</b> will be giving away ' + (Number(firstPart) && firstPart !== '1' ? '' : 'a') + ' <b>' + Tools.escapeHTML(this.prize) + '</b>!<br/>' +
```

Comment:
Now that I think about it, every Pokemon's plural is the same as the singular. Some Pokemon species still sound ok with a -s suffix (Dittos, Gligars, etc.) and some do not (Darmanitan, Gyarados, Ralts).

But really, that's just semantics of the message format and not actually related to your code.